packages/flutter/lib/src/services/keyboard_maps.dart

```diff
@@ -1493,3 +1493,230 @@ const Map<int, PhysicalKeyboardKey> kLinuxToPhysicalKey = <int, PhysicalKeyboard
   0x00000080: PhysicalKeyboardKey.showAllWindows,
 };
 
+/// Maps Web-specific key codes to the matching [LogicalKeyboardKey].
```

Comment:
Check out the comment at the top of the file: this is a generated file, generated from the Chrome source code.  See https://github.com/flutter/flutter/tree/master/dev/tools/gen_keycodes/README.md

Reply:
The comment was collapsed with the copyright comment so I didn't notice it :(

Thanks for pointing this out. I modified the code gen tool to produce the mappings for the web. Please take another look.